src/renderer/vt/ut_lib/vt.unittest.vcxproj

```diff
@@ -18,5 +18,5 @@
   <!-- DONT ADD NEW FILES HERE, ADD THEM TO vt-renderer-common.vcxitems -->
   <!-- Careful reordering these. Some default props (contained in these files) are order sensitive. -->
   <Import Project="$(SolutionDir)src\common.build.post.props" />
-  <!-- <Import Project="$(SolutionDir)src\common.build.tests.props" /> -->
+  <Import Project="$(SolutionDir)src\common.build.tests.props" />

```

Comment:
I don't know why but I need this to make CI and my local VS happy. Wonder why master is still green without it